config/default_records/submission_templates/010_rvi_bait_capture_library_submission_templates.yml

```diff
@@ -4,7 +4,8 @@
 Limber-Htp - RVI - Bait Capture Library:
```

Comment:
Just wondering whether the submission template name should be changed too. Normally, the submission template name includes the kind of sequencing.

Reply:
Good point. Ill contact the stakeholders and ask.

Reply:
Updated here and in the int suite.